src/sidebarReference.json

```diff
@@ -10,6 +10,24 @@
       "title": "Overview",
       "path": "/reference/react"
     },
+    {
+      "title": "Rules of React",
+      "path": "/reference/rules",
+      "routes": [
+        {
+          "title": "Components and Hooks must be pure",
+          "path": "/reference/rules/components-and-hooks-must-be-pure"
+        },
+        {
+          "title": "React orchestrates Components and Hooks",
+          "path": "/reference/rules/react-orchestrates-components-and-hooks"
+        },
+        {
+          "title": "Rules of Hooks",
+          "path": "/reference/rules/rules-of-hooks"
```

Comment:
Where the rules of hooks unlisted before?

Reply:
Seems like we never had them on the new docs site, only in the legacy one

Reply:
There wasn't a nav item for them, but there is an error page for them that is linked to.

The page is here: https://react.dev/warnings/invalid-hook-call-warning